scripts/README.md

```diff
@@ -6,8 +6,7 @@
 
 The [Dockerfile](./Dockerfile) might be handy for the following cases:
 
-- when you want to use not yet release (dev) version of `starport`
-- when you don't want to setup starport or swagger dependencies locally by some reason
+- when you don't want to setup Ignite or Swagger dependencies locally by some reason
 
```

Comment:
Did we re-generate openapi? Is it the same as before (no changes in snakecase or similar)?

Reply:
Yes, but new version differs
Generation scrips also modified